index2.html

```diff
@@ -0,0 +1,67 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>my second work</title>
+    <link rel="stylesheet" href="files/style2.css">
+    <meta charset="UTF-8">
+    <meta name="description" content="html, webpage, design">
+    <meta name="keywords" content="html, css, webpage, tutorial">
+    <meta name="author" content="Gunay">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  </head>
+  <body>
+    <div class="nav">
+      <div class="logo">
+        <img src="files/lg.png" alt="logo"/>
+      </div>
+      <ul>
+      <li><a href="#">Home</a></li>
+      <li><a href="#">About</a></li>
+      <li><a href="#">Products</a></li>
+      <div class="dropdown">
+        <li><button class="dropbtn">Services</button></li>
+        <div class="dropdown-content">
+          <ul>
+          <li><a href="#">Link 1</a></li>
+          <li><a href="#">Link 2</a></li>
+          <li><a href="#">Link 3</a></li>
+        </ul>
+        </div>
+      </div>
+      </ul>
+      <form class="myform" action="/action_page.php">
+        <button type="submit">Search</button>
+        <input type="text" placeholder="Search.." name="searchplc">
+      </form>
+    </div>
+    <div class="headerbackground">
```

Comment:
Please use `-` or `_` between words in CSS class naming
```suggestion
    <div class="header_background">
```